docs/api/v3.rst

```diff
@@ -346,7 +346,7 @@ Version detail
 Version update
 ++++++++++++++
 
-.. http:patch:: /api/v3/projects/(string:project_slug)/version/(string:version_slug)/
+.. http:patch:: /api/v3/projects/{project}/versions/{version}
```

Comment:
you shouldn't remove the types

Reply:
should it go like ``(string:project_slug)/versions/(string:version_slug)/`` or ``(string:{projects})/versions/(string:{version})/

Reply:
Like the first one